Examples/Algorithms/TrackFinding/src/TrackFindingOptions.cpp

```diff
@@ -1,6 +1,6 @@
 // This file is part of the Acts project.
 //
-// Copyright (C) 2020 CERN for the benefit of the Acts project
+// Copyright (C) 2020-2021 CERN for the benefit of the Acts project
```

Comment:
No other changes in this file?

Reply:
Ah, yes. There were some change but then reverted. Fixed now.